implementation/src/main/java/io/smallrye/config/WithDefault.java

```diff
@@ -18,5 +18,5 @@
      *
      * @return the default value as a string
      */
-    String value();
+    String value() default "";
```

Comment:
Yes, I had the same question, but since `""` is handled as an empty value, when you forget to add the value to the annotation, you get an error on that property name.

I wanted to avoid adding a new annotation. Maybe a `@WithDefaults` is acceptable.

Reply:
Better to see the issue when you actually code rather than seeing the issue when you start your app. 

I think we should avoid introducing a usability regression for a niche feature.

Reply:
Yes, I will try it. I'm back working on this PR but I have a few additional issues that we might have to talk about.

I'll report back soon.

Reply:
Sure. No rush :)

Reply:
Added the @WithDefaults annotation.